nodejs/awsx/ec2/vpc.ts

```diff
@@ -373,10 +391,10 @@ export interface VpcArgs {
     subnets?: VpcSubnetArgs[];
 
     /**
-     * The maximum number of availability zones to use in the current region.  Defaults to '2' if
-     * unspecified.
+     * The maximum number of availability zones to use in the current region.  Defaults to `2` if
+     * unspecified.  Use `"all"` to use all the availability zones in the current region.
```

Comment:
while i'd like to have 'undefined' mean 'use all', that could be deeply impactful to customers who would have their VPC change from 2 AZs to some other value.  So i've made this opt-in on the part of the user.  We can consider if that's the right default.

Reply:
FWIW - I actually think "2" is the right default here.  There is increased cost, and significantly diminishing returns, to using more than 2 AZs.

Reply:
ah ok.  Will stick with the this pattern then.